src/ucp/api/ucp_def.h

```diff
@@ -731,6 +731,14 @@ typedef struct ucp_ep_params {
      */
     ucs_sock_addr_t         sockaddr;
 
+    /**
+     * Local address in the form of a sockaddr; specifies the associated
+     * network device for the new connection. This setting is optional.
+     * To enable it, the corresponding - @ref
+     * UCP_EP_PARAM_FIELD_LOCAL_SOCK_ADDR bit in the field mask must be set.
+     */
+    ucs_sock_addr_t         local_sockaddr;
```

Comment:
it is really should target_sockaddr and initiator_sockaddr ... @yosefe did we release UCX with the above `sockaddr` field ?

Reply:
"sockaddr" was released a long ago

Reply:
So we ended up with quite a bit awkward naming scheme ...

Reply:
I like the description from UCT API: `The sockaddr to bind locally.`  `specifies the associated network device to bind locally to establish new connections`. We should add a reference to function for queuing the socket address

Reply:
changed accordingly